environment.yml

```diff
@@ -5,13 +5,21 @@ channels:
 - defaults
 - bioconda
 - biocore
+- anaconda
 dependencies:
-- scikit-bio=0.4.2
-- click=6.7
-- python=3
+- scikit-bio
+- click=6.7 # install from anaconda channel
+- python=3.5
 - numpy
 - scikit-learn=0.19.1
 - scipy
 - pandas
 - jupyter
 - matplotlib
+- gneiss
```

Comment:
Can you specify the version of gneiss you are using?  We may not guarantee backwards compatibility in the future.